tracer/test/Datadog.Trace.Tests/Telemetry/Metrics/common_metrics.json

```diff
@@ -601,6 +601,8 @@
         "is_unsupported_ci",
         "is_benchmark",
         "is_new",
+        "is_rum",
+        "browser_driver",
```

Comment:
Have these been added to the dd-go intake?

Reply:
yup, already there, I'm not the first one adding this feature :)